tools/VscPreviewExtension/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "DfmPreview",
-  "displayName": "DfmPreview",
+  "name": "DocFXPreview",
+  "displayName": "DocFXPreview",
   "description": "Preview Html and token tree generated by DocFX in Visual Studio Code",
-  "version": "0.0.21",
+  "version": "0.0.20",
```

Comment:
0.0.22

---
In reply to: [111536159](https://github.com/dotnet/docfx/pull/1547#discussion_r111536159) [](ancestors = 111536159)